### PR TITLE
Add the latest commit ID to builds

### PR DESCRIPTION
**Pull request by eatyourgreens**
Add the HEAD_COMMIT environment variable to the webpack build and print it out, along with build environment and date, in the index page HTML.

### CODE REVIEW

**Comment by eatyourgreens** [comment]
Jenkins built and deployed this to https://pandora.zooniverse.org in the time that it took me to create the PR. 👍

**Comment by eatyourgreens** [comment]
Screenshot of the page HTML, showing the build details as HTML comments.

**Comment by eatyourgreens** [comment]
The build environment should be staging. Other than that, this looks good.

File: package.json [hunk 1]
```diff
@@ -64,11 +64,13 @@
   "license": "Apache-2.0",
   "repository": "https://github.com/zooniverse/pandora",
   "scripts": {
-    "start": "BABEL_ENV=development webpack-dashboard -p 3001 -- webpack-dev-server --config ./webpack.config.js",
+    "_start": "NODE_ENV=staging BABEL_ENV=development webpack-dashboard -p 3001 -- webpack-dev-server --config ./webpack.config.js",
+    "start": "export HEAD_COMMIT=$(git rev-parse --short HEAD); npm run _start;",
```
Comment (camallen) [inline package.json:68]: could we inject this at build time via jenkinsfile https://github.com/zooniverse/pandora/blob/f1def66cfe21202fbd2222c7a8b0f54478e7ee24/Jenkinsfile#L22

and use a default env var with this fallback to git cmd? 
```suggestion
    "start": "export HEAD_COMMIT=${HEAD_COMMIT:-$(git rev-parse --short HEAD)}; npm run _start;",
```

**Review by camallen — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (eatyourgreens) [reply package.json:68]: I think that would work, with build args in the Dockerfile?

I copied the approach here from the PFE package file.
https://github.com/zooniverse/Panoptes-Front-End/blob/173d7502fb0802e3cdf63b90ca5e4691822b87c9/package.json#L131

**Comment by eatyourgreens** [comment]
Woo! The published commit on https://pandora.zooniverse.org updated to `6a38f25` after that commit.

File: package.json (hunk 1 above)
Comment (eatyourgreens) [reply package.json]: Force-pushed over that to alter the `build` script too. `start` is the local development server.

**Pull request merged**